### PR TITLE
serverinfo command

### PR DESCRIPTION
**Pull request by BluewyDev**
Please do let me know things i gotta fix or you want added

### CODE REVIEW

**Comment by Bloxigamer** [comment]
Hello

File: src/commands/utility/serverinfo.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  
+    static names = ['si', 'server'];
+
+    static usage = '';
+
+    static description = 'Show the servers info';
+
+    async execute() {
+        const guild = this.message.guild;
+        const embed = new Discord.MessageEmbed()
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:15]:
```suggestion
        const embed = new Discord.MessageEmbed();
```

File: src/commands/utility/serverinfo.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            embed.addField(
+                '__**Generic**__',
+                `**Owner:** ${guild.owner}\n**Owner ID:**\n${guild.ownerID}\n**Created:** ${guild.createdAt.toDateString()}\n**Region:** ${guild.region}\n**Guild ID:** ${guild.id}`,
+                true
+                );
+            embed.addField(
+                '__**Statistics**__',
+                `**Members:** ${guild.memberCount}\n**Max members:** ${guild.maximumMembers}\n**Verified:** ${guild.verified ? 'yes' : 'no'}\n**Partnered:** ${guild.partnered ? 'yes' : 'no'}`,
+                true
+            )
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:26]:
```suggestion
            );
```

File: src/commands/utility/serverinfo.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    static description = 'Show the servers info';
+
+    async execute() {
+        const guild = this.message.guild;
+        const embed = new Discord.MessageEmbed()
+            embed.setAuthor(`Info of ${guild.name}`, guild.iconURL());
+            embed.addField(
+                '__**Generic**__',
+                `**Owner:** ${guild.owner}\n**Owner ID:**\n${guild.ownerID}\n**Created:** ${guild.createdAt.toDateString()}\n**Region:** ${guild.region}\n**Guild ID:** ${guild.id}`,
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:19]: You could first create a string (`let generic = '';`) and then assign each line separately (e.g. generic += `**Owner:** ${guild.owner} \n`;) 
then you can put them each in a separate line

Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:19]: guild.owner returns a GuildMember object: https://discord.js.org/#/docs/main/stable/class/Guild?scrollTo=owner

**Review by JulianVennen — CHANGES_REQUESTED** [review]

File: src/commands/utility/serverinfo.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                true
+                );
+            embed.addField(
+                '__**Statistics**__',
+                `**Members:** ${guild.memberCount}\n**Max members:** ${guild.maximumMembers}\n**Verified:** ${guild.verified ? 'yes' : 'no'}\n**Partnered:** ${guild.partnered ? 'yes' : 'no'}`,
+                true
+            );
+            await message.channel.send(embed);
```
Comment (JulianVennen) [inline src/commands/utility/serverinfo.js:27]:
```suggestion
            await this.message.channel.send(embed);
```

**Comment by BluewyDev** [comment]
Refer to my newest pr for the serverinfo command

**Pull request closed without merging**